### PR TITLE
Update minimal burn commitment.

### PR DESCRIPTION
**Pull request by diwakergupta**

### CODE REVIEW

File: src/pages/stacks-blockchain/mining.md [hunk 1]
```diff
@@ -56,7 +56,9 @@ The miner who is selected to mine the next block is chosen depending on the amou
 
 The probability for a miner to mine next block equals the BTC miner send devided by the total BTC all miners send.
 
-The minimum amount to sent is 20,000 sats. To calculate the amount of BTC to send miners should:
+While there is no minimum BTC commitment enforced by the protocol, in practice, miners need to commit at least 11,000 satoshis: this is because each transaction includes two UTXO outputs for Proof-of-Transfer, and UTXOs smaller than 5500 satoshis run the risk of being treated as "[dust](https://unchained-capital.com/blog/dust-thermodynamics/)".
```
Comment (agraebe) [inline src/pages/stacks-blockchain/mining.md:59]: Would it make sense to briefly explain UTXOs? If the intended audience will understand, that's fine and we should be clear about that assumption

File: src/pages/stacks-blockchain/technical-specs.md [hunk 2]
```diff
@@ -16,8 +16,12 @@ description: Summary of technical specifications of Stacks 2.0
   - 500 STX/block for following 4 years
   - 250 STX/block for subsequent 4 years
   - 125 STX/block in perpetuity after that
+- Coinbase rewards accumulate for "missed sortitions": If a Bitcoin block has no sortition (at height N), then any Stacks block mined in a subsequent sortition that builds off of any Stacks chain tip that existed at the penultimate sortition (at height N-1) may claim its coinbase. This encourages miners to keep mining even if Bitcoin fees are high.
+- Initial mining bonus: This is a special case of the above to incentivize early miners. Coinbase for all burnchain blocks between the first burn block height (to be chosen by independent miners as part of the Stacks 2.0 launch) and the first sortition winner accumulate and are distributed to miners over a fixed window (to be determined). For instance, say burn block height is 10000 and first sortition is at block 10500 and distribution window is 100 blocks, then coinbase for the first 500 blocks (10500 - 10000) will be distributed evenly to miners who win sortition over the subsequent 100 blocks.
```
Comment (agraebe) [inline src/pages/stacks-blockchain/technical-specs.md:20]: `10000` > `10,000` (in several places in the para)

**Review by agraebe — APPROVED** [review]
🙏 approving so you can merge once you had a look at the comments/nits

File: src/pages/stacks-blockchain/mining.md (hunk 1 above)
Comment (diwakergupta) [reply src/pages/stacks-blockchain/mining.md:59]: A full-explanation of UTXOs is out of scope for this doc IMO. I will include a pointer to https://learnmeabitcoin.com/technical/utxo instead.

File: src/pages/stacks-blockchain/technical-specs.md (hunk 2 above)
Comment (diwakergupta) [reply src/pages/stacks-blockchain/technical-specs.md:20]: Sorry, are you saying we prefer "10000" over "10,000" or the other way around?

**Pull request merged**